config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] For best practices and clean code, kindly make sure you remove all the commented and unused code lines from your code base